packages/remote-config/lib/index.js

```diff
@@ -47,6 +47,10 @@ const statics = {
 const namespace = 'remoteConfig';
 const nativeModuleName = 'RNFBConfigModule';
 
+function isBooleanValue(v) {
+  return v === 'true' || v === 'false' || v === null || v === '0' || v === '1';
```

Comment:
```suggestion
  return v === 'true' || v === 'false' || v === '0' || v === '1';
```

Null not a boolean + also already excluding null in the if statement below.